cmd/upgrade_test.go

```diff
@@ -0,0 +1,53 @@
+package cmd
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type fakeCLI struct {
+	Version       string
+	UpgradeCalled bool
+}
+
+func (fc *fakeCLI) IsUpToDate() (bool, error) {
+	ok := fc.Version == "2.0.0"
```

Comment:
I'd suggest simplifying even further.

```
fc.UpToDate = true
```

We don't really need to deal in real version numbers—that's kind of a red herring.